### PR TITLE
readme: change images based on github theme

### PR DESCRIPTION
**Pull request by sloorush**
### What is changed and how it works?

Changes images that were ineligible when using GitHub in dark mode.

1. cncf logo
image

2. TiKV software stack
image

What's Changed:
Changed images to be shown with respect to the theme the user is in, as explained in [github documentation](https://docs.github.com/en/github/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to)

1. in dark mode shows [this image](https://raw.githubusercontent.com/sloorush/tikv/fix-dark-mode-img/images/cncf-white.png#gh-dark-mode-only) 
image
2. in dark mode shows [this image](https://raw.githubusercontent.com/sloorush/tikv/fix-dark-mode-img/images/tikv_stack_white_background.png#gh-dark-mode-only)
image

### Check List 

Tests 

- No code

Side effects

- none

### Release note 

```release-note
none
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ The design of TiKV ('Ti' stands for titanium) is inspired by some great distribu
 
 If you're interested in contributing to TiKV, or want to build it from source, see [CONTRIBUTING.md](./CONTRIBUTING.md).
 
-![cncf_logo](images/cncf.png)
+![cncf_logo](images/cncf.png#gh-light-mode-only)
```
Comment (BusyJay) [inline README.md:15]: How about using logos from https://www.cncf.io/brand-guidelines/#logo?

Comment (sloorush) [reply README.md:15]: I have used the logo mentioned in the "Variations" section in the brand guidelines for the dark mode, light mode logo is the same. 

Should I change the dark mode logo to the primary one with a white background?

Comment (BusyJay) [reply README.md:15]: Oh, what I mean is using the web image directly instead of downloading it into the repository.

Comment (sloorush) [reply README.md:15]: Oh okay! Makes sense, I'll do that.

Comment (sloorush) [reply README.md]: @BusyJay using the theme toggle needs the images to be hosted on GitHub as mentioned [here](https://stackoverflow.com/a/70200610/13308517)

Adding an external URL renders both the images in both the themes, like this:
image

Comment (sloorush) [reply README.md:15]: Reverted it back to the original(logos downloaded in the repository)

Comment (BusyJay) [reply README.md:15]: Got it.

File: README.md [hunk 2]
```diff
@@ -60,7 +61,8 @@ You can view the list of [TiKV Adopters](https://tikv.org/adopters/).
 
 ## TiKV software stack
 
-![The TiKV software stack](images/tikv_stack.png)
+![The TiKV software stack](images/tikv_stack.png#gh-light-mode-only)
```
Comment (BusyJay) [inline README.md:64]: How about using the same white background images?

Comment (sloorush) [reply README.md]: Updated it to use a single image with white background.

**Review by BusyJay — APPROVED** [review]

**Comment by sloorush** [comment]
Hey @sunxiaoguang, PTAL. 😅

**Review by youjiali1995 — APPROVED** [review]

**Pull request merged**